ragaai_catalyst/guard_executor.py

```
@@ -2,26 +2,44 @@
+            if output_deployment_id:
+                self.output_deployment_id = output_deployment_id
+                self.output_deployment_details = self.guard_manager.get_deployment(output_deployment_id)
+            if input_deployment_id and output_deployment_id:
+                # check if 2 deployments are mapped to same dataset
+                pass
+            for guardrail in self.input_deployment_details['data']['guardrailsResponse']:
+                maps = guardrail['metricSpec']['config']['mappings']
+                for _map in maps:
+                    if _map['schemaName']=='Response':
+                        raise ValueError('Response field mapping found in input guardrails')
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)